config-utilities.js

```diff
@@ -18,29 +20,35 @@ const getEnvironment = (hostname = '', subDomain = '') => {
 	return 'local';
 };
 
-class Config {
-	constructor(values) {
-		this.values = values;
+const hasConfig = () => Object.keys(cachedConfig).length > 0;
+
+const GetValue = (key) => {
```

Comment:
I know this isn't Metro, but are we doing PascalCase for public methods now?

Reply:
I checked airbnb style guide, which we are using, and they should be lower case. Will update.